### DISCUSSION TITLE
Welcome to account-aggregator-standards Discussions!

### DISCUSSION THREAD

**Discussion by dheerajkhardwal** [Sahamati/account-aggregator-standards#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by psuruli** [comment]
What is the timezone followed in Account Aggregator IST or UTC?

**Reply by dheerajkhardwal** [reply]
@psuruli Thanks for reaching out. 
The API specification follows [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) standard for date and time. And the UTC timezone is the norm in the AA ecosystem, per general observations.
